### DISCUSSION TITLE
Welcome to v2ray-heroku Discussions!

### DISCUSSION THREAD

**Discussion by zizifn** [zizifn/edgetunnel#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vargajoey6-debug** [comment]
Ok ok gotcha

**Reply by vargajoey6-debug** [reply]
Sorry newbie but for a newbie on my own I'm learning as I go verry hard

**Reply by vargajoey6-debug** [reply]
And I've done more work than u could imagine
